lib/concerns/overloadable_functions.rb

```diff
@@ -23,7 +23,7 @@ module OverloadableFunctions
   def compatible_characteristics?(service_chars, vehicle_chars)
     # Compatile service and vehicle
     # if the vehicle cannot serve the service due to sticky_vehicle_id
-    return false if !service_chars[:v_id].empty? && (service_chars[:v_id] & vehicle_chars[:id]).empty?
+    return false if !service_chars[:v_id].empty? && (service_chars[:v_id] & vehicle_chars[:v_id]).empty?
```

Comment:
this was a voluntary change, this should have been fixed on the optim-api side before starting to use this version of the gem. mt-web already passed to usage "id" we cannot modify the public api